pkg/cp/cmp/cmp.go

```diff
@@ -37,6 +37,9 @@ func isEqualFile(fpath1, fpath2 string) error {
 	return nil
 }
 
+// For testing purpose
+var readDirName = readDirNames
```

Comment:
hold on, we don't want to do this, let's focus on the watchdoc package, all of us, and get to a pattern of code we're comfortable with, then revisit the other PRs. A bit of time to slow down and get this in a good state now will be a real help later. Let's get the foundation right